src/message-list/MessageList.js

```diff
@@ -54,12 +54,12 @@ export default class MessageList extends React.PureComponent {
   }
 
   render() {
-    const { messages, subscriptions, caughtUp, fetching,
+    const { messages, caughtUp, fetching,
       fetchOlder, fetchNewer, hideFetchingOlder } = this.props;
     let messageList = [];
     let containerStyle = {};
 
-    if (messages.length === 0 || subscriptions.length === 0) {
+    if (messages.length === 0) {
```

Comment:
We should make sure all the data is fetched before showing messages otherwise the streams won't have colors